README.md

```diff
@@ -5,8 +5,8 @@
 
 ## What is it
 
-ATLauncher is a launcher for Minecraft which integrates multiple different modpacks to allow you to download and install
-modpacks easily and quickly.
+ATLauncher is a launcher for Minecraft which integrates multiple different mod-packs to allow you to download and install
```

Comment:
modpacks is already correct here. There is no dash in any platform that uses the word, so modpacks is what I've used to keep in line with every other platform